document/core/util/bikeshed_fixup.py

```diff
@@ -42,18 +38,6 @@ def Main():
         '<h3>A.' + str(number) + ' ' + section + '</h3>')
     number += 1
 
-
-  # Drop spurious navigation.
-  data = data.replace(
```

Comment:
Do you know what this was for?

Reply:
This is redundant with what appears to be an updated version at https://github.com/WebAssembly/spec/blob/main/document/core/util/mathjax2katex.py#L158
If you delete that, it does leave a spurious navigation section; I forget exactly where, but presumably comes from sphinx.